continuous_integration/travis/install.sh

```diff
@@ -56,6 +56,7 @@ conda install -q -c conda-forge \
 conda install -q -c conda-forge \
     distributed \
     cloudpickle \
+    graphviz \
```

Comment:
I think that your question remains valid.  I was getting `png` failures from graphviz.  I suspect that the defaults channel build is less-than-optimal again.  I didn't want to deal with it just now though so punted to use conda-forge for the time being.  I'll report upstream after the current build-storm passes if the problem persists.